### PR TITLE
Update 2018-07-24

### PR DESCRIPTION
**Pull request by RobertFischer**
Just ran the `./update.js` out of curiosity, and did in fact get some changes.

### CODE REVIEW

**Comment by stephenmathieson** [comment]
Hmm, maybe @TooTallNate's cron job hasn't run yet. I'd rather let it do its job, but if this is holding you up, I'll merge.

**Comment by TooTallNate** [comment]
I think something is up with my cronjob. I'll have to look into it. Thanks for the PR!

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules/
+yarn.lock
```
Comment (morissetteremi07-star) [inline .gitignore:2]: - [ ]

File: index.js
Comment (morissetteremi07-star) [inline index.js:1]: 1. @stephenmathieson @ 

1. > ### ### ### _#``_``

**Pull request merged**